component/pom.xml

```diff
@@ -0,0 +1,204 @@
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>${version.commons-logging}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.analytics-common</groupId>
+            <artifactId>org.wso2.carbon.event.input.adapter.core</artifactId>
+            <version>${version.org.wso2.carbon.analytics-common}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.rabbitmq</groupId>
+            <artifactId>amqp-client</artifactId>
+            <version>${version.com.rabbitmq}</version>
+        </dependency>
+    </dependencies>
+    <properties>
+        <version.org.wso2.carbon>4.4.3</version.org.wso2.carbon>
+        <version.org.apache.log4j.wso2>1.2.17.wso2v1</version.org.apache.log4j.wso2>
+        <version.commons-logging>1.1.3</version.commons-logging>
+        <version.org.wso2.carbon.analytics-common>5.0.9</version.org.wso2.carbon.analytics-common>
+        <version.org.wso2.cep>4.0.0</version.org.wso2.cep>
+        <version.org.apache.felix>1.7.4</version.org.apache.felix>
+        <version.com.rabbitmq>2.0.0</version.com.rabbitmq>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+                <version>${version.org.apache.felix}</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <id>generate-scr-descriptor</id>
+                        <goals>
+                            <goal>scr</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>org.wso2.carbon.extension.analytics.receiver.rabbitmq</Bundle-SymbolicName>
+                        <Bundle-Name>org.wso2.carbon.extension.analytics.receiver.rabbitmq</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal,
+                            org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal.*
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal,
+                            !org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal.*,
```

Comment:
Export packages should have version specified as a best practice. Please fix these type of issues throughout in all your PR's.